src/main/java/uk/gov/hmcts/reform/em/annotation/service/impl/AnnotationServiceImpl.java

```diff
@@ -64,6 +78,20 @@ public AnnotationDTO save(AnnotationDTO annotationDTO) {
                 }
             });
         }
+        if (annotationDTO.getTags() != null) {
+            annotation.getTags().forEach(t -> {
+                if (t.getId() == null) {
+                    tagService.persistTag(t);
+                    t.setId(UUID.randomUUID());
+                }
+
+                if (t.getAnnotation() == null) {
+                    t.setAnnotation(annotation);
```

Comment:
still not sure why we set the annotation on the tag

Reply:
it is required for mapping